app/views/notifications/_badgeachievement.html.erb

```diff
@@ -10,9 +10,9 @@
 
     <div class="crayons-card crayons-card--secondary p-4 w-100 grid gap-2">
       <img class="w-100 max-w-50 l:max-w-25 h-auto inline-block mx-auto" src="<%= optimized_image_url(json_data["badge_achievement"]["badge"]["badge_image_url"], width: 250) %>" alt="<%= json_data["badge_achievement"]["badge"]["title"] %>">
+  <% end %>
       <p class="color-base-70"><em><%= json_data["badge_achievement"]["rewarding_context_message"].html_safe %></em></p>
     </div>
```

Comment:
Okay, very important to note here that I am putting this `end` BEFORE a `<div>` closes properly. My gut says this is a bad idea, BUT (big but!) it worked when I tested it... This allows us to cache the image as well as the title.

Could definitely use a gut check here. I think the alternative is to move the `<p>` tag outside of the `<div>` but would want the design team's input before adjusting that.